src/get-files-stream.js

```diff
@@ -31,7 +30,7 @@ function getFilesStream (files, opts) {
       adder.add(vinylfs.src(file, srcOpts))
 
       // if recursive, glob the contents
-      if (opts.r || opts.recursive) {
+      if (opts.recursive) {
```

Comment:
I'm merging the `r` option in the configuration setup, so later down the line it's only one option to check.